test/test-versions.js

```diff
@@ -101,23 +151,28 @@ function testVersions(filename) {
 
   /**
    * @param {Identifier} data
-   * @param {string} relPath
+   * @param {string} [relPath]
+   * @param {string} [category]
    */
-  function findSupport(data, relPath) {
+  function findSupport(data, relPath, category) {
     for (const prop in data) {
       if (prop === '__compat' && data[prop].support) {
-        checkVersions(data[prop].support, relPath, logger);
+        checkVersions(data[prop].support, relPath, category, logger);
       }
       const sub = data[prop];
       if (typeof sub === 'object') {
-        findSupport(sub, relPath ? `${relPath}.${prop}` : `${prop}`);
+        findSupport(
+          sub,
+          relPath ? `${relPath}.${prop}` : `${prop}`,
+          category ? category : prop,
+        );
       }
     }
   }
   findSupport(data);
 
   if (errors.length) {
-    console.error(chalk`{red   Versions – }{red.bold ${errors.length}}{red  ${errors.length === 1 ? 'error' : 'errors'}:}`);
+    console.error(chalk`{red   Versions – {bold ${errors.length}} ${errors.length === 1 ? 'error' : 'errors'}:}`);
```

Comment:
I'm talking about things like this.  Changes like this one that do not adjust the functionality, especially based upon the title of the PR, should be in separate PRs.